### PR TITLE
Address Component dropdown Fix

### PR DESCRIPTION
**Pull request by nasar-cambria**
Which User story does this relate to?
https://osi-cwds.atlassian.net/browse/CALS2-5067

Brief feature description
Address component state dropdown in wood-duck was not retaining selection - Internet Explorer

Does this PR block/effect other stories?
-no

Do any commands needs to be run in order to run the project with this PR?
-no

Code coverage percent diff as a result of this PR?

### CODE REVIEW

**Comment by edgibbs** [comment]
Can one of the admins verify this patch?

File: src/CommonAddressComponent.js [hunk 1]
```diff
@@ -5,11 +5,11 @@ import DropDownField from './DropDownField.js';
 import ReactAutosuggest from 'react-autosuggest';
 
 const getDictionaryId = object => {
-  return (object && object.id) || '';
+  return object === null ? '' : object.value;
```
Comment (ratneshraval) [inline src/CommonAddressComponent.js]: This can still be written as `(object && object.value) || ''`. Was that not working?

**Comment by ratneshraval** [comment]
Is this build stuck? Not getting test results status.

File: src/CommonAddressComponent.js (hunk 1 above)
Comment (nasar-cambria) [reply src/CommonAddressComponent.js]: Thanks @ratneshraval Updated.

**Comment by edgibbs** [comment]
The Taborda server is full and we haven't had time to migrate the builds.  CodeClimate has two silly dup issues so it should be fine.

**Comment by edgibbs** [comment]
Had @alyssaburkecwds approve the storybook story, and given the limited scope she was fine with it.  The CodeClimate build actually has no issues, despite the message, and the Taborada Jenkins server is still down and we are migrating to the state CI.

**Pull request merged**